testapp/java/org/openyolo/testapp/AuthenticationMethodsInputView.java

```diff
@@ -103,7 +103,7 @@ private void initialize(Context context) {
         }
 
         if (mIdAndPasswordCheckBox.isChecked()) {
-            authenticationMethods.add(AuthenticationMethods.ID_AND_PASSWORD);
+            authenticationMethods.add(AuthenticationMethods.EMAIL);
         }
 
```

Comment:
Doesn't have to be in this cl, but we should also update this view to reflect the new well defined authentication methods.

However we should update the UI for at least this checkbox field from "openyolo://id-and-password" to "openyolo://email"